### DISCUSSION TITLE
Removing a character from a string under edit?

### DISCUSSION THREAD

**Discussion by MiqViq** [Spirik/GEM#91] [General]
When editing a string, it seems there is no way to delete a single char within a string (at start/between/at the end), thus shortening the string.

**Comment by Spirik** [comment]
Yes, that is unfortunately true: GEM offers only "in-place" edition of characters in a string (and adding new at the very end) - you can not delete character from within a string with the rest of the string shifted, nor you can insert a character in the arbitrary place in a string (with the rest shifted).
